tests/devices/test_default_qubit_tf.py

```diff
@@ -1329,23 +1329,18 @@ def test_sample_observables_non_differentiable(self):
         shots = 100
         dev = qml.device("default.qubit.tf", wires=2, shots=shots)
 
-        @qml.qnode(dev, diff_method="backprop", interface="tf")
-        def circuit(a):
-            qml.RX(a, wires=0)
-            return qml.sample(qml.PauliZ(0))
-
-        a = tf.Variable(0.54)
+        with pytest.raises(qml.QuantumFunctionError, match="Devices with finite shots"):

```

Comment:
Should we keep this test? Maybe changing the description to better fit the error message?

Reply:
Since we test elsewhere that this error is raised, I'm deleting this test.